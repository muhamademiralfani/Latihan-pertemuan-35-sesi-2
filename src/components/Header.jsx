import React from 'react'

const Header = () => {
  return (
    <h1>Welcome to my App</h1>
  )
}

export default Header