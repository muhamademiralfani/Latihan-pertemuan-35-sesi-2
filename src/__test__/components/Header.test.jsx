/* eslint-disable no-undef */
import { render } from '@testing-library/react';
import Header from '../../components/Header';

describe('Header component', () => {
  test('should render the Header component', () => {
    render(<Header />);
    expect(document.querySelector('h1')).toBeInTheDocument();
  });

  test('match snapshot', () => {
    const { container } = render(<Header />);
    expect(container).toMatchSnapshot();
  });
});
