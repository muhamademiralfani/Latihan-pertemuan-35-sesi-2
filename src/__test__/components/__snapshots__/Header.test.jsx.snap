// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component match snapshot 1`] = `
<div>
  <h1>
    Welcome to my App
  </h1>
</div>
`;
